### PR TITLE
Use <a> instead of window.open

### PR DESCRIPTION
**Pull request by T-Damer**
- Explain what this PR changes
- in one or more
- bullet points

### CODE REVIEW

File: src/components/navbar/Wallet.tsx [hunk 1]
```diff
@@ -69,28 +68,33 @@ export default function () {
...
-              'No wallet connected'
-            )}
-          </AccentText>
-        </div>
-        <div className={width('w-fit')}>
-          <SealWallet connected={!!account} />
+        <div
+          className={accountLinkContainer}
+          onClick={async () => {
+            if (!account) return await WalletStore.connect(true)
```
Comment (MixailE) [inline src/components/navbar/Wallet.tsx:79]: Can we simplify it like that `account && WalletStore.connect(true)`?

**Review by MixailE — APPROVED** [review]

File: src/components/navbar/Wallet.tsx [hunk 2]
```diff
@@ -69,28 +68,33 @@ export default function () {
...
       )}
-      <div
-        className={accountLinkContainer}
-        onClick={async () => {
-          if (!account) return await WalletStore.connect(true)
-          openNewTabAndFocus(getEtherscanAddressUrl(account, Network.Goerli))
-        }}
+      <a
+        href={
+          account ? getEtherscanAddressUrl(account, Network.Goerli) : undefined
```
Comment (smyzin) [inline src/components/navbar/Wallet.tsx:73]: Don't we need to place it inside `div.accountLinkContainer`?

**Review by smyzin — APPROVED** [review]
Lgtm

File: src/components/navbar/Wallet.tsx (hunk 1 above)
Comment (T-Damer) [reply src/components/navbar/Wallet.tsx:79]: Ah, yes

File: src/components/navbar/Wallet.tsx (hunk 2 above)
Comment (T-Damer) [reply src/components/navbar/Wallet.tsx]: This will break the layout

**Pull request merged**